upstream-community-operators/credstash-operator/1.13.0/credstash-operator.v1.13.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,257 @@
+    ```
+    AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY
+    AWS_REGION
+    ```
+    For example running the following will create an appropriate secret in the credstash namespace;
+    `kubectl create secret generic aws-credentials --from-literal=AWS_ACCESS_KEY_ID=access_key --from-literal=AWS_SECRET_ACCESS_KEY=secret_access_key --from-literal=AWS_REGION=us-west-2 --namespace=placeholder`
+    ### Example Credstash Secrets
+    #### Opaque
+    ```
+    apiVersion: credstash.ouzi.tech/v1alpha1
+    kind: CredstashSecret
+    metadata:
+      name: very-secret
+      namespace: test
+    spec:
+      # Name of the target secret (Optional. Defaults to the CR name)
+      name: example
+      # List of secrets from credstash to add to the body of the secret
+      secrets:
+          # key: the key in credstash to fetch. (Required)
+        - key: test-secret
+          # name: the name of the resulting data element in the k8s secret (Optional. Defaults to the credstash key)
+          name: renamed-test-secret
+          # table: the dynamoDB table that contains the credstash secrets (Optional. Defaults to credential-store)
+          table: credential-store
+          # version: the version of the secret in credstash for the provided key (Optional. Defaults to the latest version)
+          version: 1
+          # context: key value pairs to the encryption context (Optional)
+          context:
+            key1: value1
+            key2: value2
+      # type: the type of the resulting kubernetes secret (Optional. Defaults to Opaque)
+      type: Opaque
+    ```
+    #### Docker Config
+    ```
+    apiVersion: credstash.ouzi.tech/v1alpha1
+    kind: CredstashSecret
+    metadata:
+      name: dockerconfigjson
+      namespace: test
+    spec:
+      # Name of the target secret (Optional. Defaults to the CR name)
+      name: dockerconfigjson
+      # List of secrets from credstash to add to the body of the secret
+      secrets:
+          # key: the key in credstash to fetch. (Required)
+        - key: docker_secret
+          # name: the name of the resulting data element in the k8s secret (Optional. Defaults to the credstash key)
+          name: .dockerconfigjson
+          # table: the dynamoDB table that contains the credstash secrets (Optional. Defaults to credential-store)
+          table: credential-store
+          # context: key value pairs to the encryption context (Optional)
+          context:
+            key1: value1
+            key2: value2
+      # type: the type of the resulting kubernetes secret (Optional. Defaults to Opaque)
+      type: kubernetes.io/dockerconfigjson
+    ```
+    ### Documentation
+    For further Documentation, visit the [credstash-operator repo in github](https://github.com/ouzi-dev/credstash-operator)
+
+  maturity: alpha
+  version: 1.13.0
+  replaces: ''
```

Comment:
Since this is first version of operator. Please remove `replaces`. It should be used only when you update to new version. Thanks